### ISSUE TITLE
Missing core/intel.cc and core/processors.cc

### ISSUE THREAD

**Issue by HappyDOGE** [Alukym/VMProtect-Source#1]
Hi!
The leak has some essential files missing.
Does anyone know if they have been published at all?
The original BBS thread was deleted so maybe they were posted in the original leak.

**Comment by HugoMskn** [comment]
Keep me updated if you find anything

**Comment by ztgasdf** [comment]
I'm gonna get on a list for posting here but fuck it: Looks like whoever leaked it either intentionally didn't leak it or whatever they obtained didn't come with it in the first place.

> 关键的实现部分 arm跟intel 没发

**Comment by HugoMskn** [comment]
Anyone has a screen or copy of the thread ?

**Comment by HappyDOGE** [comment]
![image](https://github.com/Alukym/VMProtect-Source/assets/28511119/773c93d8-0978-4dab-a48e-bf605fa4bf28)
**The vmpsrc.zip file was repacked.**
The original vmpsrc archive is a split ZIP (source: Google cache). It is unknown whether or not anyone downloaded it.

**Comment by HugoMskn** [comment]
that's the zip here

[vmpsrc.zip](https://github.com/Alukym/VMProtect-Source/files/11466610/vmpsrc.zip)

**Comment by HugoMskn** [comment]
If someone have the og download link maybe it's still on the servers

**Comment by HappyDOGE** [comment]
No, your file is a unified ZIP, while the OG thread had split ZIP. If the archive was repacked, this could mean that the missing files were removed by the person who uploaded the unified ZIP.

**Comment by HugoMskn** [comment]
Who made the og post, this github user ?

**Comment by ztgasdf** [comment]
@HappyDOGE Do you have a link to the original thread, even though it's deleted?

**Comment by HappyDOGE** [comment]
The original archive was posted at https://bbs.kanxue.com/thread-277192.htm

**Comment by HugoMskn** [comment]
not on webarchive machine

**Comment by HugoMskn** [comment]
OH WAIT ITS ON WEBARHCIVe https://web.archive.org/web/20230512112041/https://bbs.kanxue.com/thread-277192.htm

**Comment by HugoMskn** [comment]
OG Poster : https://bbs.kanxue.com/homepage-242832.htm

**Comment by HappyDOGE** [comment]
> OH WAIT ITS ON WEBARHCIVe https://web.archive.org/web/20230512112041/https://bbs.kanxue.com/thread-277192.htm
unfortunately only the edited post is in webarchive, with attachments gone

**Comment by HugoMskn** [comment]
Maybe they are still on the server, but  i cant download shit

**Comment by 3e08488f9bcc2dd50192b3db89f6612c** [comment]
me when sherlock failed download a public zip file.

**Comment by arzam16** [comment]
Made a torrent (hybrid v1+v2) of this repo and whatever was on the anonfiles. Both archives don't have `core/intel.cc` and `core/processors.cc`, though.

```
magnet:?xt=urn:btih:47e63ebfaceeadef42bab937ea60f2a5544729c5&xt=urn:btmh:12203a656593298416737f106866e351276d89b44662cf04610dd8e4a16a7c9c4317&dn=VMProtect%20Source
```

also announcing to `best` trackers listed in https://github.com/ngosang/trackerslist

**Comment by HugoMskn** [comment]
anyone tried this ?
![image](https://github.com/Alukym/VMProtect-Source/assets/81116453/195d69a8-a78b-4363-af26-4ef3938562ca)

**Comment by HappyDOGE** [comment]
You can't download it without either having WeChat or a Mainland China phone number

**Comment by HugoMskn** [comment]
I contacted a chineese friend

**Comment by gir489returns** [comment]
arm.cc is also missing for targeting ARM binaries.

**Comment by invlpg** [comment]
> I contacted a chineese friend

same the cheese people have it

**Comment by invlpg** [comment]
![image](https://github.com/Alukym/VMProtect-Source/assets/75157064/7077b27c-003d-4bfe-863b-0830ec4b6155)

**Comment by invlpg** [comment]
> You can't download it without either having WeChat or a Mainland China phone number

![image](https://github.com/Alukym/VMProtect-Source/assets/75157064/e794bd02-7888-48e1-98b7-0cb12ff96954)

**Comment by XutaxKamay** [comment]
> anyone tried this ? ![image](https://user-images.githubusercontent.com/81116453/238065444-195d69a8-a78b-4363-af26-4ef3938562ca.png)

got it from a friend, files aren't inside sadly

**Comment by invlpg** [comment]
intel.cc is like 30 thousand lines of code???

**Comment by XutaxKamay** [comment]
> intel.cc is like 30 thousand lines of code??? no comments???
that's literally the most interesting inside yeah lol

**Comment by XutaxKamay** [comment]
processor.cc would be nice too

**Comment by invlpg** [comment]
processor.cc and intel.cc stay in mainland china with me

**Comment by XutaxKamay** [comment]
> processor.cc and intel.cc stay in mainland china with me

why do you keep them in hostages?
poor little guys.

**Comment by invlpg** [comment]
VMprotect软件太混乱，不值得您花时间

https://www.youtube.com/watch?v=qGk4E9ss95s

**Comment by invlpg** [comment]
从前有一个叫李明的男人，他是一名天才的计算机程序员。李明在软件保护领域有着卓越的技术才能，他帮助许多公司保护了他们的软件免受盗版和破解的侵害。

然而，李明内心深处有一种渴望，他想要分享自己的技术成果，让更多的人受益。他觉得软件保护应该是一个开放的领域，大家可以共同进步。然而，这个想法却与当时的政治环境相悖。

有一天，李明得到了一份文件，其中包含了中国最高级别的软件保护技术——VMProtect的源代码。这是一个绝对机密的文件，只有少数人掌握。李明陷入了艰难的抉择：他应该保守这个秘密还是将其分享给其他人。

最终，李明决定违抗规定，泄露了VMProtect的源代码。他相信这样做可以推动软件保护技术的发展，使更多人受益。然而，不久之后，中国政府发现了这个泄露事件。

中共当局认为李明的行为是对国家安全的威胁，对他展开了调查。他被捕并被判处终身监禁。李明的家人和朋友都为他感到心痛和不解。他们认为李明只是出于善意，希望贡献自己的智慧和技术。

在监狱中，李明度过了余生。他思考着自己的决定是否正确，他是否应该听从内心的声音。尽管面临着终身监禁的命运，李明从未后悔过他的选择。他相信，尽管他被囚禁，但他的行动将会为软件保护技术的发展带来积极的影响。

多年后，李明的事迹渐渐被人们所了解。他的勇气和追求科技进步的精神被人们所赞赏。他成为了一代人心中的英雄，尽管他的遭遇令人心痛。

这个故事提醒我们，技术和政治之间有时候存在冲突。我们每个人都有权利追求知识和技术进步，但我们也需要考虑行动的后果以及所处的环境。

**Comment by nuukedo29** [comment]
> 



> 从前有一个叫李明的男人，他是一名天才的计算机程序员。李明在软件保护领域有着卓越的技术才能，他帮助许多公司保护了他们的软件免受盗版和破解的侵害。
> 
> 然而，李明内心深处有一种渴望，他想要分享自己的技术成果，让更多的人受益。他觉得软件保护应该是一个开放的领域，大家可以共同进步。然而，这个想法却与当时的政治环境相悖。
> 
> 有一天，李明得到了一份文件，其中包含了中国最高级别的软件保护技术——VMProtect的源代码。这是一个绝对机密的文件，只有少数人掌握。李明陷入了艰难的抉择：他应该保守这个秘密还是将其分享给其他人。
> 
> 最终，李明决定违抗规定，泄露了VMProtect的源代码。他相信这样做可以推动软件保护技术的发展，使更多人受益。然而，不久之后，中国政府发现了这个泄露事件。
> 
> 中共当局认为李明的行为是对国家安全的威胁，对他展开了调查。他被捕并被判处终身监禁。李明的家人和朋友都为他感到心痛和不解。他们认为李明只是出于善意，希望贡献自己的智慧和技术。
> 
> 在监狱中，李明度过了余生。他思考着自己的决定是否正确，他是否应该听从内心的声音。尽管面临着终身监禁的命运，李明从未后悔过他的选择。他相信，尽管他被囚禁，但他的行动将会为软件保护技术的发展带来积极的影响。
> 
> 多年后，李明的事迹渐渐被人们所了解。他的勇气和追求科技进步的精神被人们所赞赏。他成为了一代人心中的英雄，尽管他的遭遇令人心痛。
> 
> 这个故事提醒我们，技术和政治之间有时候存在冲突。我们每个人都有权利追求知识和技术进步，但我们也需要考虑行动的后果以及所处的环境。

因此，我们无非可以学习李明对于软件保护领域的付出，同时也通过这些经历，可以理解如何做到对技术、社ety和地方改变的合作，从而实现良好的社区效acy，同时也符合每个角色在这个关系中所扮演的角色。

**Comment by XutaxKamay** [comment]
https://www.youtube.com/watch?v=g3jCAyPai2Y

**Comment by invlpg** [comment]
当中共来到我家，让我一辈子去劳教所时，我会记住你们兄弟

**Comment by nuukedo29** [comment]
https://github.com/Alukym/VMProtect-Source/assets/55753080/60de327f-4716-4996-b677-12f6ab4432a4

**Comment by nuukedo29** [comment]
https://github.com/Alukym/VMProtect-Source/assets/55753080/6a3a3180-bf29-4173-a2e0-d05ed3b33334

**Comment by nuukedo29** [comment]
https://github.com/Alukym/VMProtect-Source/assets/55753080/d1e7f406-2c5b-4232-8969-89e814e6ad7d

**Comment by invlpg** [comment]
+rep

**Comment by gir489returns** [comment]
动态网自由门 天安門 天安门 法輪功 李洪志 Free Tibet 六四天安門事件 The Tiananmen Square protests of 1989 天安門大屠殺 The Tiananmen Square Massacre 反右派鬥爭 The Anti-Rightist Struggle 大躍進政策 The Great Leap Forward 文化大革命 The Great Proletarian Cultural Revolution 人權 Human Rights 民運 Democratization 自由 Freedom 獨立 Independence 多黨制 Multi-party system 台灣 臺灣 Taiwan Formosa 中華民國 Republic of China 西藏 土伯特 唐古特 Tibet 達賴喇嘛 Dalai Lama 法輪功 Falun Dafa 新疆維吾爾自治區 The Xinjiang Uyghur Autonomous Region 諾貝爾和平獎 Nobel Peace Prize 劉暁波 Liu Xiaobo 民主 言論 思想 反共 反革命 抗議 運動 騷亂 暴亂 騷擾 擾亂 抗暴 平反 維權 示威游行 李洪志 法輪大法 大法弟子 強制斷種 強制堕胎 民族淨化 人體實驗 肅清 胡耀邦 趙紫陽 魏京生 王丹 還政於民 和平演變 激流中國 北京之春 大紀元時報 九評論共産黨 獨裁 專制 壓制 統一 監視 鎮壓 迫害 侵略 掠奪 破壞 拷問 屠殺 活摘器官 誘拐 買賣人口 遊進 走私 毒品 賣淫 春畫 賭博 六合彩 天安門 天安门 法輪功 李洪志 Winnie the Pooh 劉曉波动态网自由门

**Comment by gir489returns** [comment]
> chiner_1.mp4

Damm, China is alright.

**Comment by moeray** [comment]
> ![image](https://user-images.githubusercontent.com/28511119/238043174-773c93d8-0978-4dab-a48e-bf605fa4bf28.png) **The vmpsrc.zip file was repacked.** The original vmpsrc archive is a split ZIP (source: Google cache). It is unknown whether or not anyone downloaded it (the cache shows least 2 downloads, so it was likely downloaded).

Key files were missing from the original archive, and the leaker deleted them on purpose, maybe they want to resell them again

**Comment by HappyDOGE** [comment]
@moeray do you have the original split archive? if so, can you upload it somewhere

**Comment by moeray** [comment]
> @moeray do you have the original split archive? if so, can you upload it somewhere

[2023-05.zip](https://github.com/Alukym/VMProtect-Source/files/11468866/2023-05.zip)

**Comment by invlpg** [comment]
> > ![image](https://user-images.githubusercontent.com/28511119/238043174-773c93d8-0978-4dab-a48e-bf605fa4bf28.png) **The vmpsrc.zip file was repacked.** The original vmpsrc archive is a split ZIP (source: Google cache). It is unknown whether or not anyone downloaded it (the cache shows least 2 downloads, so it was likely downloaded).
> 
> Key files were missing from the original archive, and the leaker deleted them on purpose, maybe they want to resell them again

we keep it private hehehe

**Comment by HappyDOGE** [comment]
Very cool!

**Comment by FarmEquipment69** [comment]
![Xinnie the Pooh](https://upload.wikimedia.org/wikipedia/commons/b/b5/Xinnie_the_Pooh.png)

**Comment by HappyDOGE** [comment]
@invlpg can you prove you have the files?

**Comment by Space-CN-G** [comment]
> @invlpg can you prove you have the files?

It's nothing but an anti-China robot that, upon triggering keywords, automatically responds.

**Comment by HappyDOGE** [comment]
Looks like it, but I'll give him the benefit of the doubt.

**Comment by MurrayCapricorns** [comment]
Actually,The leaked's editor told that there was not intel.cc orginally.which means is even split or not split both are same.

**Comment by HappyDOGE** [comment]
Yes, unfortunately the original split ZIP (or at least the split ZIP posted by @moeray above) does not have the files as well.

**Comment by 3e08488f9bcc2dd50192b3db89f6612c** [comment]
![image](https://github.com/Alukym/VMProtect-Source/assets/69107484/9966730f-e242-4e59-854e-899add59713c)
looks like your reading sucks a lot.

**Comment by 3e08488f9bcc2dd50192b3db89f6612c** [comment]
It tells that Item and arm are not released if you are curious,

**Comment by Doliman100** [comment]
> anyone tried this ? ![image](https://user-images.githubusercontent.com/81116453/238065444-195d69a8-a78b-4363-af26-4ef3938562ca.png)

This is the same file as on anonfiles.com, just packaged in 7z.
![vmpsrc_2](https://github.com/Alukym/VMProtect-Source/assets/5512376/ee9ffabf-6572-4634-a54a-9fd172039210)

**Comment by moeray** [comment]
The author of vmp breathed a faint sigh of relief, lol

**Comment by HugoMskn** [comment]
How long for ? what if we get a pdb and get the compilled version and try extracting it ?

**Comment by aureliancnx** [comment]
no

**Comment by invlpg** [comment]
> > @invlpg can you prove you have the files?
> 
> It's nothing but an anti-China robot that, upon triggering keywords, automatically responds.

😢 i am from china

**Comment by gd1551** [comment]
> @invlpg can you prove you have the files?

well, can you?

**Comment by invlpg** [comment]
> @invlpg can you prove you have the files?

intel.cc has 31934 lines of code and processors.cc has 2850 lines

**Comment by HappyDOGE** [comment]
well, that's not exactly proof (but interesting information). can you show a small code snippet that follows the same codestyle as the rest of the codebase? some class function that is mentioned in intel headers or something

**Comment by aureliancnx** [comment]
> > @invlpg can you prove you have the files?
> 
> intel.cc has 31934 lines of code and processors.cc has 2850 lines

false, processors.cc has 3011 lines, intel.cc 33198

**Comment by HappyDOGE** [comment]
well there is no way to prove either of these claims so you both might be wrong

**Comment by ztgasdf** [comment]
just upload it :/

**Comment by invlpg** [comment]
> well, that's not exactly proof (but interesting information). can you show a small code snippet that follows the same codestyle as the rest of the codebase? some class function that is mentioned in intel headers or something

![AMyjhwPA](https://github.com/Alukym/VMProtect-Source/assets/75157064/c7a7a778-f65a-41b4-878a-962f8ead2ef8)

**Comment by invlpg** [comment]
> well, that's not exactly proof (but interesting information). can you show a small code snippet that follows the same codestyle as the rest of the codebase? some class function that is mentioned in intel headers or something
![proof](https://github.com/Alukym/VMProtect-Source/assets/75157064/58d1672a-327b-4777-b0a6-db5c0ebaaa0c)

**Comment by moeray** [comment]
> > @invlpg can you prove you have the files?
> 
> intel.cc has 31934 lines of code and processors.cc has 2850 lines

Has core/resource. h also been deleted? Why?

**Comment by SquareTetrisBlock** [comment]
![82a](https://github.com/Alukym/VMProtect-Source/assets/88846317/a8e738a8-e7c0-4ff3-a5e1-76f4ce3c4025)
